src/object.rs

```diff
@@ -327,11 +327,13 @@ impl<'a> OccupiedEntry<'a> {
         }
     }
     /// Returns a reference to the key at this entry
+    #[must_use]
     pub fn key(&self) -> &IString {
         self.get_key_value().0
     }
 
     /// Removes and returns the entry as a (key, value) pair.
+    #[must_use]
```

Comment:
These `#[must_use]` annotations seem over-zealous. For example, the corresponding method in `libstd` does not have `#[must_use]`. Which clippy lint is requiring this?

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#must_use_candidate

Reply:
OK, I think this is a false positive though. `remove_entry` mutates the underlying map and should not be marked `must_use`.